app/importers/file_importers/courses_sections_importer.rb

```diff
@@ -0,0 +1,29 @@
+class CoursesSectionsImporter < Struct.new :school_scope, :client, :log, :progress_bar
+
+  def remote_file_name
+    'courses_sections_export.txt'
+  end
+
+  def data_transformer
+    CsvTransformer.new
+  end
+
+  def filter
+    SchoolFilter.new(['HEA', 'WSNS', 'ESCS', 'BRN', 'KDY', 'AFAS', 'WHCS', 'SHS'])
+  end
+
+  def school_ids_dictionary
+    @dictionary ||= School.all.map { |school| [school.local_id, school.id] }.to_h
+  end
+
+  def import_row(row)
+    course = CourseRow.new(row, school_ids_dictionary).build
+    if course.school.present?
```

Comment:
I've been able to resolve the data issues with some fixes to queries. Would still rather not fail hard since it is simply course data. Will add a bit more logging.